html5lib/__init__.py

```diff
@@ -19,7 +28,8 @@
 from .serializer import serialize
 
 __all__ = ["HTMLParser", "parse", "parseFragment", "getTreeBuilder",
-           "getTreeWalker", "serialize"]
+           "getTreeWalker", "serialize", "__version__"]
 
 # this has to be at the top level, see how setup.py parses this
+#: Distribution version number, which asymptotically approaches 1.
```

Comment:
I'd remove this. Amongst other things, it's not going to be true for much longer.

Reply:
Thank goodness! Will do.